src/clusterfuzz/_internal/platforms/android/symbols_downloader.py

```diff
@@ -179,6 +180,35 @@ def download_system_symbols_if_needed(symbols_directory):
   utils.write_data_to_file(build_params, build_params_check_path)
 
 
+def download_trusty_symbols_if_needed(symbols_directory, app_name):
+  """Downloads and extracts Trusted App ELF files"""
+  ab_target = ''
+  device = settings.get_build_parameters().get('target')
+  if device in ['cheetah', 'panther']:
```

Comment:
As a future improvement: would it make sense to encode these details as part of the build rather than hardcode things thing?

Reply:
Yes, I definitely agree! - encoding these details would allow this code to continue working as device types and branch names are added or changed. I'm not sure I can personally make that change, but I can bring this up with Trusty team.